x-pack/plugins/infra/public/pages/logs/log_entry_rate/sections/anomalies/table.tsx

```diff
@@ -240,18 +201,99 @@ export const AnomaliesTable: React.FunctionComponent<{
     ],
     [collapseId, expandId, expandedIds, dateFormat]
   );
+  return (
+    <>
+      <EuiBasicTable
+        items={tableItems}
+        itemId="id"
+        itemIdToExpandedRowMap={expandedDatasetRowContents}
+        isExpandable={true}
+        hasActions={true}
+        columns={columns}
+        sorting={tableSortOptions}
+        onChange={handleTableChange}
+      />
+      <EuiSpacer size="l" />
+      <PaginationControls
+        fetchNextPage={fetchNextPage}
+        fetchPreviousPage={fetchPreviousPage}
+        page={page}
+        isLoading={isLoading}
+      />
+    </>
+  );
+};
+
+const AnomalyMessage = ({
+  actual,
+  typical,
+  type,
+}: {
+  actual: number;
+  typical: number;
+  type: string;
+}) => {
+  const messageType = type === 'logRate' ? 'dataset' : 'category';
+  const moreThanExpectedAnomalyMessage = i18n.translate(
+    'xpack.infra.logs.analysis.anomaliesTableMoreThanExpectedAnomalyMessage',
+    {
+      defaultMessage: 'higher log messages in this {messageType} than expected',
+      values: { messageType },
+    }
+  );
+
+  const fewerThanExpectedAnomalyMessage = i18n.translate(
+    'xpack.infra.logs.analysis.anomaliesTableFewerThanExpectedAnomalyMessage',
+    {
+      defaultMessage: 'fewer log messages in this {messageType} than expected',
+      values: { messageType },
+    }
+  );
+  const isMore = actual > typical;
+  const message = isMore ? moreThanExpectedAnomalyMessage : fewerThanExpectedAnomalyMessage;
+  const ratio = isMore ? actual / typical : typical / actual;
+  const icon = isMore ? 'sortUp' : 'sortDown';
+  // Edge case scenarios where actual and typical might sit at 0.
+  const useRatio = ratio !== Infinity;
+  const ratioMessage = useRatio ? `${formatOneDecimalPlace(ratio)}x` : '';
 
   return (
-    <EuiBasicTable
-      items={pageOfItems}
-      itemId="id"
-      itemIdToExpandedRowMap={expandedDatasetRowContents}
-      isExpandable={true}
-      hasActions={true}
-      columns={columns}
-      pagination={paginationOptions}
-      sorting={sorting}
-      onChange={handleTableChange}
-    />
+    <span>
+      <EuiIcon type={icon} /> {`${ratioMessage} ${message}`}
+    </span>
+  );
+};
+
+const PaginationControls = ({
+  fetchPreviousPage,
+  fetchNextPage,
+  page,
+  isLoading,
+}: {
+  fetchPreviousPage?: () => void;
+  fetchNextPage?: () => void;
+  page: number;
+  isLoading: boolean;
+}) => {
+  return (
+    <EuiFlexGroup justifyContent="center">
+      <EuiFlexItem grow={false}>
+        <EuiFlexGroup>
+          <EuiButtonIcon
+            iconType="arrowLeft"
+            isDisabled={!fetchPreviousPage || isLoading}
+            onClick={fetchPreviousPage}
+          />
+          <span>
+            <strong>{page}</strong>
+          </span>
+          <EuiButtonIcon
+            iconType="arrowRight"
+            isDisabled={!fetchNextPage || isLoading}
+            onClick={fetchNextPage}
+          />
```

Comment:
These are causing many warnings like

> EuiButtonIcon requires aria-label or aria-labelledby to be specified because icon-only

to be shown for me. Could we add the required properties?